File: Python/7kyu/0007returnAstringsEvenCharacters/rs.py
def main():
    def even_chars(st):
        if len(st) < 2 or len(st) > 100:
            return 'invalid string'
        else:
            result = []
            increment = 0
            for character in st:
                if increment % 2:
                    result.append(character)
                increment += 1
            print(result)
            return result
    even_chars("abcdefg")


if __name__ == "__main__":
    main()

# Status: Solved

File: JavaScript/7kyu/0073returnAstringsEvenCharacters/rs.js
function evenChars(string) {
  if (string.length > 100 || string.length < 2) {
    return "invalid string";
  } else {
    let result = [];
    for (let i = 1, len = string.length; i < len; i += 2) {
      result.push(string[i]);
    }
    return result;
  }
}

// Status: Solved